mkdocs.yml

```diff
@@ -211,11 +213,10 @@ plugins:
         redirect_maps:
             'docs/installing.md': 'docs/getting-started/install.en.md'
             'docs/getting-started.md': 'docs/getting-started/first-app-tutorial.en.md'
-            'docs/commands/pipeline-update.en.md': 'docs/commands/pipeline-deploy.en.md'
-            'docs/commands/pipeline-update.ja.md': 'docs/commands/pipeline-deploy.ja.md'
-            'docs/developing/additional-aws-resources.en.md': 'docs/developing/addons/workload.en.md'
-            'docs/developing/additional-aws-resources.ja.md': 'docs/developing/addons/workload.ja.md'
-            'docs/developing/service-discovery.en.md': 'docs/developing/svc-to-svc-communication.en.md'
+            'docs/commands/pipeline-update.md': 'docs/commands/pipeline-deploy.en.md'
+            'docs/developing/additional-aws-resources.md': 'docs/developing/addons/workload.en.md'
+            'docs/developing/service-discovery.md': 'docs/developing/svc-to-svc-communication.en.md'
```

Comment:
maybe leave these unchanged until fixed?

Reply:
It's all busted anyway today, it wouldn't be a regression for cusotmers 😂